content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -73,30 +80,36 @@ Do not use trailing slashes when specifying the path to the packages directory.
 
 Use CMake to build the Visual Studio project in the build directory of your O3DE project.
 
-1. Build the project launcher using the solution you created in the `build/windows_vs2019` directory. The `profile` build configuration is shown in this example.
+1. Build the project launcher using the solution you created in the project's `build/windows_vs2019` directory. The `profile` build configuration is shown in this example.
 
     ```cmd
-    cmake --build build/windows_vs2019 --target MyProject.GameLauncher --config profile -- /m
+    cmake --build build/windows_vs2019 --target MyProject.GameLauncher Editor --config profile -- /m
     ```
 
-    {{< note >}}
-The `/m` is a recommended build tool optimization, which tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
-    {{< /note >}}
+    Note that when building the project for a pre-built SDK engine, even though you are not building **O3DE Editor**, it is still recommended to include `Editor` as a build target. While the GameLauncher doesn't depend on the Editor target, there are some Gems that do. If you leave off the Editor target when building your project, those Gems won't be included in the build.
+
+    When building the project for a source engine, the **Asset Processor** and **Project Manager** are built too, since they are dependencies of **O3DE Editor**.
+
+    The `/m` is a recommended build tool optimization, which tells the Microsoft compiler (MSVC) to use multiple threads during compilation to speed up build times.
 
-1. If you're using an installed O3DE engine (or engine SDK), the project should build within minutes. In this example, when the build is complete, the project binaries can be found under `build/windows_vs2019/bin/profile` in the project directory.
+1. When the build is complete, the project binaries can be found in the project directory under `build/windows_vs2019/bin/profile`. Run **O3DE Editor** to verify the project is ready to use.
 
-1. Run the O3DE Editor to verify the project is ready to use.
+    **Scenario 1:** If you set up your engine as a [source engine](/docs/welcome-guide/setup/setup-from-github/#build-the-engine), you do not need to include the project path.
+
+    ```cmd
+    build\windows_vs2019\bin\profile\Editor.exe
+    ```
```

Comment:
It is more accurate to state that if the Editor applications is within the project directory, than the project path does not need to be supplied.

If a user were for some reason to place there project build directory outside of the project path, it would require the `--project-path` in that scenario as well

Reply:
This is great. I appreciate little tweaks like this. It's such an important topic to be accurate. I will clarify.